lib/whitehall_importer/import.rb

```diff
@@ -73,5 +74,16 @@ def first_publish_date
     def first_edition_history
       EditionHistory.new(whitehall_document["editions"].first["revision_history"])
     end
+
+    def integrity_check(document)
+      integrity_check_edition(document.live_edition) if document.live_edition
+      integrity_check_edition(document.current_edition) if document.current_edition != document.live_edition
+    end
+
+    def integrity_check_edition(edition)
+      integrity_checker = WhitehallImporter::IntegrityChecker.new(edition)
```

Comment:
You shouldn't need this to be prefixed with "WhitehallImporter::"

Reply:
Oops! Will fix.